### PR TITLE
ooniprobe 3.5.1

### PR DESCRIPTION
**Pull request by hellais**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by carlocab** [comment]
```
==> Verifying checksum for '3c2d482eeefe577194a22e7258ba9ea0860988ba9bea3fd0a35252ba16ed62f6--probe-cli-3.5.1.tar.gz'
Error: SHA256 mismatch
Expected: 0743f87646aac321440c546865839dc0724c170125f332af3551c00720a676f7
  Actual: 8278f9318ef939c46169dc44e89bbc03915c660912e66be9c4b20d18d00816fe
    File: /Users/brew/Library/Caches/Homebrew/downloads/3c2d482eeefe577194a22e7258ba9ea0860988ba9bea3fd0a35252ba16ed62f6--probe-cli-3.5.1.tar.gz
```

**Comment by carlocab** [comment]
For future reference, I think you did:
```
❯ curl https://github.com/ooni/probe-cli/archive/v3.5.1.tar.gz | sha256sum
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   122  100   122    0     0    525      0 --:--:-- --:--:-- --:--:--   525
0743f87646aac321440c546865839dc0724c170125f332af3551c00720a676f7  -
```
when what you actually want is
```
❯ curl -L https://github.com/ooni/probe-cli/archive/v3.5.1.tar.gz | sha256sum
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   122  100   122    0     0   1326      0 --:--:-- --:--:-- --:--:--  1311
100  546k    0  546k    0     0  1315k      0 --:--:-- --:--:-- --:--:-- 1315k
8278f9318ef939c46169dc44e89bbc03915c660912e66be9c4b20d18d00816fe  -
```
to get the correct checksum.

File: Formula/ooniprobe.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Ooniprobe < Formula
   desc "Network interference detection tool"
   homepage "https://ooni.org/"
-  url "https://github.com/ooni/probe-cli/archive/v3.4.0.tar.gz"
-  sha256 "e573cc6496860b75c02d35a1829c220c9c8062350f2178fce208698538bb3ced"
+  url "https://github.com/ooni/probe-cli/archive/v3.5.1.tar.gz"
+  sha256 "0743f87646aac321440c546865839dc0724c170125f332af3551c00720a676f7"
```
Comment (carlocab) [inline Formula/ooniprobe.rb:5]:
```suggestion
  sha256 "8278f9318ef939c46169dc44e89bbc03915c660912e66be9c4b20d18d00816fe"
```

**Review by carlocab — APPROVED** [review]
Thanks, @hellais!

**Comment by hellais** [comment]
@carlocab 
> when what you actually want is

Ah yes you are right, I forgot to follow the redirects on that link. Thanks for fixing the shasum!

**Comment by carlocab** [comment]
Sure. And, one more thing: please use homebrew/core style in your commit messages. Yours should've just read `ooniprobe 3.5.1`.

Thanks again!

**Comment by hellais** [comment]
> please use homebrew/core style in your commit messages

Sure thing, will keep that in mind for the future!

Any idea why the build is stuck and it's not mergeable yet?

**Comment by carlocab** [comment]
We had some long-running CI jobs (e.g. Python, go) run simultaneously, so there's currently a long CI queue. This should get done soon-ish.

**Pull request closed without merging**